addons/mrp/models/mrp_production.py

```diff
@@ -871,6 +871,9 @@ def create(self, vals_list):
                 vals['name'] = self.env['stock.picking.type'].browse(picking_type_id).sequence_id.next_by_id()
             if not vals.get('procurement_group_id'):
                 procurement_group_vals = self._prepare_procurement_group_vals(vals)
+                field = self.env['procurement.group']._fields.get('move_type')
+                if field and field.default and 'move_type' not in procurement_group_vals:
+                    procurement_group_vals['move_type'] = field.default(self.env['procurement.group'])
```

Comment:
It's a bit odd/hacky
Why not simply use `clean_context` in the appropriate place ?

Reply:
I don't really like the idea of cleaning the context because in another scenario the same line might need the context that is getting passed while its execution. So this is why I think it is better to just add the default move_type value for the procurement group in its creation.

Reply:
Default values provided in the rpc should be related to one model only (account.move in your case)
Letting the code uses these default values for another model does not make sense
You just need to find the appropriate place, so you don't remove any default values provided py side for another model